app/controllers/login/cas_controller.rb

```diff
@@ -56,8 +56,13 @@ def create
     if st.is_valid?
       reset_session_for_login
 
-      pseudonym = @domain_root_account.pseudonyms.for_auth_configuration(st.user, aac)
-      pseudonym ||= aac.provision_user(st.user) if aac.jit_provisioning?
+      sso_user = st.user
+      if !st.extra_attributes["EmpoyeeNumber"].nil? && cas_login_url.ends_with?('cas/6')
```

Comment:
should we use '''blank''' instead of nil incase we get an empty element and not just a missing element?